open-api/rest-catalog-open-api.yaml

```diff
@@ -3804,6 +4025,21 @@ components:
             }
           }
 
+    UnprocessableEntityResponse:
+      description:
+        Clients are required to do preplanning and provide a plan-task, before initiating a plan.
```

Comment:
Similar comment to PayloadTooLarge, this description should be generic. We can make the example specific to PlanTable.

Reply:
will fix